addons/mail/views/mail_templates.xml

```diff
@@ -281,6 +283,7 @@
                 <script type="text/javascript" src="/mail/static/src/messaging/component/follower_list_menu/follower_list_menu_tests.js"></script>
                 <script type="text/javascript" src="/mail/static/src/messaging/component/follower_subtype/follower_subtype_tests.js"></script>
                 <script type="text/javascript" src="/mail/static/src/messaging/component/partner_im_status_icon/partner_im_status_icon_tests.js"></script>
+                <!--<script type="text/javascript" src="/mail/static/src/messaging/component/partner_mention_suggestion/partner_mention_suggestion_tests.js"></script>-->
```

Comment:
to remove comment or remove line, depending on what is the goal of the change

Reply:
Done